packages/expo-notifications/android/src/main/java/expo/modules/notifications/service/NotificationsService.kt

```diff
@@ -134,7 +142,7 @@ open class NotificationsService : BroadcastReceiver() {
     fun handleResponseReceived(context: Context, response: NotificationResponse, receiver: ResultReceiver? = null) {
       val data = getUriBuilderForIdentifier(response.notification.notificationRequest.identifier).appendPath("response").build()
       doWork(context, Intent(NOTIFICATION_EVENT_ACTION, data).also { intent ->
-        intent.putExtra(EVENT_TYPE_KEY, RECEIVE_TYPE)
+        intent.putExtra(EVENT_TYPE_KEY, RECEIVE_RESPONSE_TYPE)
         intent.putExtra(NOTIFICATION_RESPONSE_KEY, response)
```

Comment:
can we remove this function now? It looks like this is called from `NotificationsHelper.responseReceived`, but all those calls were in the `NotificationResponseReceiver` and `TextInputNotificationResponseReceiver`